quodlibet/po/fr.po

```diff
@@ -5195,11 +5176,11 @@ msgstr ""
 
 #: ../quodlibet/qltk/prefs.py:319
 msgid "_Fixed image filename:"
-msgstr ""
+msgstr "Nom de l'image fixe :"
```

Comment:
English descriptions could be better .. but I'd say 
msgid "_Fixed image filename:" -> "Fichier de jaquettes unique"
msgid "The single image filename to use if selected" -> "Sélectionne la jaquette en fonction de ce nom de fichier uniquement"
msgid "The album art image file to use when forced" -> "Fichier image, du répertoire courant, de jaquette à utiliser"
? 
Could add that else it uses "filename_front.jpg, _back, _cover, _folder ..." but it isn't in the english version. Also, not clear (from my tests) if embedded art is preferred when "Fixed image filename" is checked..